types/audio-type/index.d.ts

```diff
@@ -0,0 +1,17 @@
+declare module 'audio-type' {
+  export default function audioType(buf: Uint8Array | ArrayBuffer | null): string | undefined;
+  
+  export function isMp3(buf: Uint8Array | ArrayBuffer | null): boolean;
+
+  export function isWav(buf: Uint8Array | ArrayBuffer | null): boolean;
+
+  export function isOgg(buf: Uint8Array | ArrayBuffer | null): boolean;
+
+  export function isFlac(buf: Uint8Array | ArrayBuffer | null): boolean;
+
+  export function isM4a(buf: Uint8Array | ArrayBuffer | null): boolean;
+
+  export function isOpus(buf: Uint8Array | ArrayBuffer | null): boolean;
+
+  export function isQoa(buf: Uint8Array | ArrayBuffer | null): boolean;
+}
```

Comment:
`declare module` isn’t necessary when the file is only for one module:

```suggestion
export default function audioType(buf: Uint8Array | ArrayBuffer | null): string | undefined;

export function isMp3(buf: Uint8Array | ArrayBuffer | null): boolean;

export function isWav(buf: Uint8Array | ArrayBuffer | null): boolean;

export function isOgg(buf: Uint8Array | ArrayBuffer | null): boolean;

export function isFlac(buf: Uint8Array | ArrayBuffer | null): boolean;

export function isM4a(buf: Uint8Array | ArrayBuffer | null): boolean;

export function isOpus(buf: Uint8Array | ArrayBuffer | null): boolean;

export function isQoa(buf: Uint8Array | ArrayBuffer | null): boolean;
```